spec/components/dashboard/collection_component_spec.rb

```diff
@@ -122,5 +122,9 @@
     it "renders a link to purl" do
       expect(rendered.css("a").map { |node| node["href"] }).to include "https://purl.stanford.edu/yq268qt4607"
     end
+
+    it "Copy icon for purl link has aria-label with work title" do
```

Comment:
Would you consider the more English-y

```suggestion
    it "includes an aria-label with work title" do
```

❓

Reply:
well,
- the block context is `with a work that has a druid` 
- the aria label is for the cute little copy icon, not the purl link

which is how I arrived at the verbose description.

Reply:
OK, then how about:

```suggestion
    it "renders a copy icon link with an informative aria-label" do
```

Trying to lean into RSpec description best practice to avoid gobbledygook output.

Reply:
it's not the purl link that has the aria-label.  There is a copy icon image that has the link.

Reply:
I reworded slightly.

Reply:
Do you want to come up with a concise English phrase for this test that is of the form "verbs a noun"?